c3pzero_hackathon_config/config/site_config.yaml

```
@@ -0,0 +1,9 @@
+#
+#  This contains information for a unique instance of a robot.
+#
+
+# Name of the package to specialize
+based_on_package: "c3pzero_hackathon"
```

This needs to point to the name of the base config package you're specializing:

```suggestion
based_on_package: "c3pzero_base_config"
```